### DISCUSSION TITLE
F-Droid

### DISCUSSION THREAD

**Discussion by linsui** [evan361425/flutter-pos-system#163] [General]
Could I package this app for F-Droid? Does it make sense?

**Comment by evan361425** [comment]
Sorry for late response...
It is my first time hearing about F-Droid so I take a quick tour of it.

Do you have more details about why should we publish this app on it?

**Reply by linsui** [reply]
It's an app store for FOSS apps. If a user doesn't have Google Play they can install it from F-Droid. I'm not sure if this is a demo though so I don't know if there are such users.

**Reply by evan361425** [reply]
Thanks for the explanation. I will consider deploying the app to F-Droid after the official English version is released (currently primarily in Traditional Chinese).

But you can still package it to F-Droid yourself if you want it!
